packages/config/src/memory/memory.config.ts

```diff
@@ -105,6 +105,7 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
 
   /** Find all imagery inside this configuration and create a virtual tile set for it */
   createVirtualTileSets(): void {
+    const allLayers: ConfigLayer[] = [];
```

Comment:
We need to in the future handle the removal of layers from the `aerial` tileset, so when we do that all children virutal tilesets will dissapear.

we need to retain `aerial:antipodes-islands-satellite-2019-2020-0.5m` even if it doesnt exist inside of `aerial:` 

Also how often is `aerial:...` actually used? we should get some metrics.

Reply:
Turns out it was basically just OSM using the `aerial:` prefix, we have migrated OSM to the newer direct imagery names so now there is pretty much 0 access to `aerial:` and it is safe to remove.